src/main/java/Check.java

```diff
@@ -0,0 +1,72 @@
+public class Check {
+     //Function to check if an input has mark as the first word
+     public static boolean isMark(String input){
+        if(input.length()>=4){
```

Comment:
Would be good if there is spacing among the variables and operators, noticed the same thing for the other places too
```suggestion
        if (input.length() >= 4) {
```